src/site/defaultTemplate.scala.html

```diff
@@ -1,11 +1,8 @@
-<%@ var title:String %>
-<%@ var content:String %>
-<%@ var toc:Traversable[(String,String)] %>
-<%@ var useBanner : Boolean = false %>
+@(title:String,content:String,toc:Traversable[(String,String)],useBanner:Boolean)
```

Comment:
Formatting: Spaces after commas would make this more readable if that is allowed